eip-0001.md

```diff
@@ -120,14 +120,29 @@ Specification: List Of All Proposed Endpoints
 - GET: `/scan/boxesUnspent/{scanId}` - List all boxes that have been tracked by the scan and are still unspent/part of the UTXO set
 
 
-Specification: Simple Wallet Scan
+Specification: List Of Implemented Endpoints
+--------------------------------------------
+
+- POST: `/scan/register` - 3.3.0
+- POST: `/scan/deregister` - 3.3.0
+- POST: `/scan/stopTracking/{scanId}/{boxId}` - 3.3.0
```

Comment:
It is already too late, but maybe for consistency don't pass POST parameter as part of URL?

Reply:
Actually, that's how it was implemented